### PR TITLE
add method that will create html element

### LINKED ISSUE #30: add method that will create html element
should include: 
1. type
2. id (optional)
3. classes (optional)
4. value (optional, lets say we want to create p tag/button/h3 etc... when we create div value can be added but should be IGNORED during component rendering)

example:
createElement('div', 'example-id', ['test-class', 'test-class-2', 'Hello, its me div value']

<div id="example-id" class="test-class test-class-2"><div>

### PR DESCRIPTION
**Pull request by ibednorz**
mistake again, I typed fit add . and files from ib-9 are added. Only the interface and createElement function have been added

planned extension - properties and iteration over it

### CODE REVIEW

File: src/app/interfaces/elementConfig.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export interface elementConfig {
```
Comment (bartoszbialecki) [inline src/app/interfaces/elementConfig.ts]: Name should be in PascalCase - ElementConfig

File: src/app/utils/createElement.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  const element = document.createElement(type)
+  if (content) {
+    element.textContent = content
+  }
+  if (id) {
+    element.id = id
+  }
+  if (classes) {
+    classes.forEach((className) => {
+      element.className += `${className} `
```
Comment (bartoszbialecki) [inline src/app/utils/createElement.ts]: It's ok, but you can also do: element.classList.add(className)

File: src/app/utils/createElement.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    })
+  }
+  if (handleEvent?.type) {
+    element.addEventListener(handleEvent.type, () => handleEvent.callback())
+  }
```
Comment (bartoszbialecki) [inline src/app/utils/createElement.ts:29]: What if I would like to add more event listeners? Maybe you can use here an array?

File: src/app/utils/createElement.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+export function createElement({
+  type,
+  content,
+  id,
+  classes,
+  handleEvent,
+  properties,
+  innerHTML,
+  styles,
```
Comment (bartoszbialecki) [inline src/app/utils/createElement.ts:11]: styles is not used in the function body

**Review by pdybowski — APPROVED** [review]

**Review by hoolek77 — APPROVED** [review]

**Review by pdybowski — COMMENTED** [review]
add styles

**Review by pdybowski — CHANGES_REQUESTED** [review]
comment

File: src/app/interfaces/elementConfig.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
+export interface ElementConfig {
```
Comment (pdybowski) [inline src/app/interfaces/elementConfig.ts]: Change name to customHtmlElement

File: src/app/utils/createElement.ts [hunk 6]
```diff
@@ -0,0 +1,39 @@
+import { ElementConfig } from '../interfaces/elementConfig'
```
Comment (pdybowski) [inline src/app/utils/createElement.ts]: From 'interfaces'

In interfaces folder add index.ts and inside index.ts add export * from customHtmlElement

**Review by pdybowski — CHANGES_REQUESTED** [review]
Just few things

File: src/app/components/info/index.ts [hunk 7]
```diff
@@ -0,0 +1,121 @@
+import { renderPopup } from '../popup'
+import { createElement } from '../../utils/createElement'
```
Comment (pdybowski) [inline src/app/components/info/index.ts]: Just from utils

**Review by pdybowski — APPROVED** [review]

**Review by aleksandraj036 — APPROVED** [review]

**Pull request merged**